tests/out/shadow.spvasm.snap

```diff
@@ -22,12 +22,12 @@ OpMemberName %17 1 "pos"
 OpMemberName %17 2 "color"
 OpName %19 "Lights"
 OpMemberName %19 0 "data"
-OpName %24 "c_ambient"
```

Comment:
I am confused as to why IR has changed here. The code doesn't look like it's necessarily changing the IR.
You can get the IR by running it this way:
```bash
cargo run --features wgsl-in -- tests/in/shadow.wgsl >shadow.txt
```